subgraphs/enzyme-core/utils/actionId.ts

```diff
@@ -1,20 +1,20 @@
-export const enum AaveDebtPositionActionId {
+export enum AaveDebtPositionActionId {
```

Comment:
why do we remove const?

Reply:
Mainly because it's then consistent with the other `enum` exports. Both seem to work, though.